test/models/util/amounts.spec.ts

```diff
@@ -0,0 +1,266 @@
+import {
+  getUnitPrice,
+  getBuyAmountFromPrice,
+  getUnlimitedOrderAmounts,
+} from "../../../src/util/amounts";
+import { Fraction } from "../../../src/fraction";
+import BN from "bn.js";
+import { assert } from "chai";
+import "mocha";
+
+const MAX128 = new BN(2).pow(new BN(128)).subn(1);
+const FLOAT_TOLERANCE = new BN(2).pow(new BN(52)); // This is float precision
+const ZERO_BN = new BN(0);
+const TEN_BN = new BN(10);
+const MAX8 = new BN(2).pow(new BN(8)).subn(1);
+const MAX256 = new BN(2).pow(new BN(256)).subn(1);
```

Comment:
Very much nitpick: move `MAX128` between `MAX8` and `MAX256`.
```suggestion
const FLOAT_TOLERANCE = new BN(2).pow(new BN(52)); // This is float precision
const ZERO_BN = new BN(0);
const TEN_BN = new BN(10);
const MAX8 = new BN(2).pow(new BN(8)).subn(1);
const MAX128 = new BN(2).pow(new BN(128)).subn(1);
const MAX256 = new BN(2).pow(new BN(256)).subn(1);
```
(Interestingly, Github's diff algorithm doesn't minimize the number of lines changed.)